nodeup/pkg/model/packages.go

```diff
@@ -52,6 +52,12 @@ func (b *PackagesBuilder) Build(c *fi.ModelBuilderContext) error {
 		c.AddTask(&nodetasks.Package{Name: "pigz"})
 		c.AddTask(&nodetasks.Package{Name: "socat"})
 		c.AddTask(&nodetasks.Package{Name: "util-linux"})
+		if b.Distribution == distributions.DistributionUbuntu2010 {
+			// install podman for crio testing
+			c.AddTask(&nodetasks.Package{Name: "podman"})
+		}
+		// For ARM testing
+		c.AddTask(&nodetasks.Package{Name: "runc"})
```

Comment:
We probably shouldn't install these if the container runtime isn't crio.

Reply:
Agreed and also should not install `runc`. This should be installed from the cri-o package.